FWCore/Framework/src/StreamSchedule.cc

```diff
@@ -544,26 +544,27 @@ namespace edm {
     Traits::setStreamContext(streamContext_, ep);
     Traits::preScheduleSignal(actReg_.get(), &streamContext_);
 
-    try {
-      HLTPathStatus hltPathStatus(hlt::Pass, 0);
-      for (int empty_trig_path : empty_trig_paths_) {
-        results_->at(empty_trig_path) = hltPathStatus;
-        pathStatusInserters.at(empty_trig_path)->setPathStatus(streamID_, hltPathStatus);
-        pathStatusInserterWorkers_.at(empty_trig_path)->runStatusInserter<OccurrenceTraits<EventPrincipal, BranchActionStreamBegin>>(
-            ep, es, streamID_, ParentContext(&streamContext_), &streamContext_
-        );
-      }
-      for (int empty_end_path : empty_end_paths_) {
-        endPathStatusInserterWorkers_.at(empty_end_path)->runStatusInserter<OccurrenceTraits<EventPrincipal, BranchActionStreamBegin>>(
-            ep, es, streamID_, ParentContext(&streamContext_), &streamContext_
-        );
+    HLTPathStatus hltPathStatus(hlt::Pass, 0);
+    for (int empty_trig_path : empty_trig_paths_) {
+      results_->at(empty_trig_path) = hltPathStatus;
```

Comment:
In the class HLTGlobalStatus both the operator[] and the at function call the at function of the vector contained in the class. Calling at seemed more honest. I suppose I could change HLTGlobalStatus if you want me to.
http://cmslxr.fnal.gov/source/DataFormats/Common/interface/HLTGlobalStatus.h#0058